docs/kratos/configuring.md

```diff
@@ -4,7 +4,7 @@ title: Configuring
 ---
 
```

Comment:
This file still has the `.md` extension. Please change to `.mdx`.

Reply:
I think it's a good opportunity to do this for all files. Thanks!

Reply:
See my other comment